ansible_collections/arista/avd/molecule/dhcp_provisionning/molecule.yml

```diff
@@ -30,7 +30,7 @@ driver:
   name: docker
 platforms:
   - name: DC1-LEAF1A
-    image: avdteam/base:3.6
+    image: avdteam/base:3.8
```

Comment:
We should converge all base image to the same and `3.8-v2.0` is more up-to-date than the old one

Reply:
Fixed